crypto/comp/c_brotli.c

```diff
@@ -0,0 +1,757 @@
+/*
+ * Copyright 1998-2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Uses brotli compression library from https://github.com/google/brotli
```

Comment:
Looks as though it's available on Ubuntu via `apt`. So... yes? It also means I may want to update the CI to use the package version rather than grabbing source. Zstd is also available via `apt`.

Reply:
Yes - that would be preferable.

Reply:
This latest push has those updates to the tests.